src/App.js

```diff
@@ -4,9 +4,30 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+    visibleMovies: moviesFromServer,
+  };
+
+  handleQuerySearch = (event) => {
+    const query = event.target.value;
+    const visibleMovies = moviesFromServer
+      .filter(movie => movie.title
+        .toLowerCase()
+        .includes(query.toLowerCase())
+        || movie.description
+          .toLowerCase()
+          .includes(query.toLowerCase()));
+
+    this.setState({
+      query,
+      visibleMovies,
+    });
```

Comment:
```suggestion
```
You can get rid of this value in the state and make filtering in `render` method. Then you should only pass `visibleMovies` to the `MoviesList` component